src/EdgeGPT/chathub.py

```diff
@@ -229,5 +229,22 @@ async def _initial_handshake(self, wss: aiohttp.ClientWebSocketResponse) -> None
         await wss.receive(timeout=900)
         await wss.send_str(append_identifier({"type": 6}))
 
+    async def remove_and_close(self) -> None:
```

Comment:
Would prefer to have this as `remove_conversation(conversation_id:str=None, conversation_signature:str=None,client_id:str=None)`